### DISCUSSION TITLE
Danalock V3-BTZB

### DISCUSSION THREAD

**Discussion by josefkrtil** [Koenkk/zigbee2mqtt#7738] [Devices]
Hi,

is somebody using Danalock V3-BTZB? Have you issues with battery drain? I have to change batteries every week. I use it via zigbee2mqqt and Home assistant. I use standard confuguration of exposes or longer time for exposes battery (6 hours) but the same result.

**Comment by greynoO** [comment]
I have it but was never able to integrate it properly. Interview always fails.
